src/main/scala/com/gu/contentapi/firehose/ContentApiFirehoseConsumer.scala

```diff
@@ -1,22 +1,25 @@
 package com.gu.contentapi.firehose
 
 import com.gu.contentapi.firehose.client.StreamListener
-import com.gu.contentapi.firehose.kinesis.{ KinesisStreamReader, KinesisStreamReaderConfig, SingleEventProcessor }
-import com.gu.crier.model.event.v1.EventPayload.{ Atom, UnknownUnionField }
+import com.gu.contentapi.firehose.kinesis.{KinesisStreamReader, KinesisStreamReaderConfig, SingleEventProcessor}
+import com.gu.crier.model.event.v1.EventPayload.UnknownUnionField
 import com.gu.crier.model.event.v1.EventType.EnumUnknownEventType
-import com.gu.crier.model.event.v1.{ Event, EventPayload, EventType }
+import com.gu.crier.model.event.v1.{Event, EventPayload, EventType}
 import com.twitter.scrooge.ThriftStructCodec
 import software.amazon.awssdk.auth.credentials.AwsCredentialsProvider
-import software.amazon.kinesis.lifecycle.{ ShutdownReason }
-import software.amazon.kinesis.processor.{ ShardRecordProcessor, ShardRecordProcessorFactory }
+import software.amazon.kinesis.coordinator.CoordinatorConfig
+import software.amazon.kinesis.lifecycle.ShutdownReason
+import software.amazon.kinesis.processor.{ShardRecordProcessor, ShardRecordProcessorFactory}
 
 import scala.concurrent.duration._
 
 class ContentApiFirehoseConsumer(
   val kinesisStreamReaderConfig: KinesisStreamReaderConfig,
   override val credentialsProvider: AwsCredentialsProvider,
   val streamListener: StreamListener,
-  val filterProductionMonitoring: Boolean = false) extends KinesisStreamReader {
+  val filterProductionMonitoring: Boolean = false,
+  val clientVersionCompatibility: CoordinatorConfig.ClientVersionConfig // see https://github.com/guardian/content-api-firehose-client/pull/56
```

Comment:
I haven't provided a default here, in order to force consideration in projects that use KCL in more than one place - they need to be aware that by taking this library upgrade, they're using KCL v3 everywhere, and this `ClientVersionConfig` setting should be made on the `coordinatorConfig` of _every_ KCL client that's in the project.